working/m12-strings-english.txt

```diff
@@ -8,21 +10,34 @@
-^L18^[1C FF FB 02 _L2214_][1C FF 49 00 _L2215_]^L2218^[09 FF C8 00][09 FF C9 00][1C FF 62 00 _L40_][1C FF 5F 00 _L2216_][D4 FF 0B 00][1B FF 14 00]^L2214^@Beeep...[1D FF][D4 FF 0B 00][00 FF]
+^L15^[1C FF FB 02 _L2204_][1C FF 49 00 _L2205_][1C FF 4A 00 _L2206_][83 FF D5 01][82 FF _L2207_][1C FF 64 00 _L2208_][1C FF 5E 00 _L2209_][00 FF]
+^L2204^@(I've got fleas sucking me[01 FF]  dry,[1B FF 0F 00] so I'm just going to lie[01 FF]  still for now.)[1D FF][00 FF]
+^L16^[1C FF 60 00 _L2219_][C6 FF][1B FF 1E 00][C4 FF 00 B1][1B FF 78 00][1B FF 78 00][C5 FF]@Hey![1B FF 0F 00] L-[1B FF 0F 00]L-[1B FF 0F 00]Listen to what I've[01 FF]  got to say![02 FF]@When I took Picky to the place[01 FF]  where the meteorite landed...[02 FF][DB FF 11 00 02 00 0E 00][CE FF 11 00 00 00]@Oh![1B FF 14 00]  Good evening ma'am.[1B FF 0F 00][01 FF]  You're looking lovely as usual.[01 FF]  *snicker*[02 FF][DB FF 11 00 02 00 0E 00][CE FF 11 00 00 00]@Anyway,[1B FF 0F 00] as I was saying,[02 FF]@the police that were guarding[01 FF]  the meteorite landing left[01 FF] suddenly[02 FF]@to deal with the Sharks.[1B FF 14 00] You[01 FF]  know the Sharks,[1B FF 0A 00] they're the[01 FF]  local ruffians,[02 FF]@and they were really going[01 FF]  wild.[1B FF 14 00] Suddenly,[1B FF 14 00] I noticed that[01 FF]  Picky was gone.[02 FF]@I blame the cops... it certainly[01 FF]  wasn't my fault at all.[02 FF]@When my dad gets back,[1B FF 0F 00] I[01 FF]  know I'm gonna get it...[02 FF]@You're my bestest friend.[1B FF 0F 00][01 FF]  Won't YOU help me find Picky?[02 FF][22 FF][03 FF 00 00][04 FF _L2220_][05 FF _L2221_][00 FF]
+^L2221^@If you refuse me,[1B FF 0F 00] I'll[01 FF]  say something that'll[01 FF]  cut you like a knife.[02 FF]@Will you come with me?[01 FF][22 FF][03 FF 00 00][04 FF _L2220_][05 FF _L2222_]
+^L2222^@Okay, okay... I won't say[01 FF]  anything to hurt you.[02 FF]@Will you please[01 FF]  come with me?[01 FF][22 FF][03 FF 00 00][04 FF _L2220_][05 FF _L2221_][80 FF _L2221_]
+^L2220^[08 FF 60 00]@Ok,[1B FF 0F 00] good buddy![1B FF 14 00] Let's blow[01 FF]  this popsicle stand...[02 FF]@Before we go,[1B FF 0F 00] why don't you[01 FF]  say <goodbye> to your mom.[DB FF 11 00 02 00 0E 00][CE FF 11 00 00 00][1B FF 0A 00][01 FF]  Don't you agree,[1B FF 0F 00] ma'am?[A1 FF 0E 00 0D 00][1D FF][00 FF]
+^L17^[1C FF 49 00 _L2229_][1C FF 73 02 _L2230_][1C FF 4C 00 _L2231_]@I'll do everything I can to help[01 FF]  you![1B FF 14 00]  Good luck on your[01 FF]  adventure.[02 FF]@You might get hungry along the[01 FF]  way,[1B FF 0F 00] so here is a Cookie.[22 FF][91 FF FF][82 FF _L2232_][93 FF 00 54][08 FF 4C 00][86 FF _L2233_][02 FF]@You can also leave anything[01 FF]  you don't need with me.[02 FF]@Do you want me to hang on to[01 FF]  something?[01 FF][03 FF 00 00][04 FF _L2234_][05 FF _L2235_][00 FF]
+^L2235^@Be careful![1D FF][00 FF]
+^L2234^[86 FF _L2236_][00 FF]
+^L18^[1C FF FB 02 _L2214_][1C FF 49 00 _L2215_][00 FF]

```

Comment:
This 00 FF is what made the Dad call at the beginning of the game not happen, if you wanted to try to continue further.

Reply:
Fixed.